app/components/Header/DropdownContents/LoginDropdown.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import {
+  LoginForm,
+  RegisterForm,
+  ForgotPasswordForm,
+} from 'app/components/LoginForm';
+import styles from '../Header.css';
+
+type Props = {};
+
+type State = {
+  mode: string,
+};
+
+const LoginDropdown = (props: Props, state: State) => {
+  const isLogin = state.mode === 'login';
+  let title, form;
+
+  state = {
+    mode: 'login',
+  };
+
+  switch (state.mode) {
+    case 'login':
+      title = 'Logg inn';
+      form = <LoginForm />;
+      break;
+    case 'register':
+      title = 'Register';
+      form = <RegisterForm />;
+      break;
+    case 'forgotPassword':
+      title = 'Glemt passord';
+      form = <ForgotPasswordForm />;
+      break;
+    default:
+      break;
+  }
+
+  return (
+    <div className={styles.eventsDropdownEl}>
+      <div className={styles.dropdownSection} data-first-dropdown-section>
+        {title}
+        <button
+          onClick={this.toggleForgotPassword}
+          className={styles.toggleButton}
+        >
+          Glemt passord
+        </button>
+        <span className={styles.toggleButton}>&bull;</span>
```

Comment:
What is `&bull;`?

Reply:
You'll have to ask @SmithPeder 🙃 But ye, it's a "dot". I plan on redesigning this Login dropdown anyways, so it will soon cease to exist :)